flexmeasures/ui/utils/view_utils.py

```diff
@@ -18,7 +18,7 @@
 
 def render_flexmeasures_template(html_filename: str, **variables):
     """Render template and add all expected template variables, plus the ones given as **variables."""
-    variables["flask_env"] = current_app.env
+    variables["flexmeasures_env"] = current_app.env
```

Comment:
Is `app.env` actually still being used after this PR? If you look at `app.py`, it seems it's not. Maybe you can check what this (if you render a page).

Follow-up thoughts:
- Is this `flask_env` variable being used in any template? I couldn't find any use?
- If `app.env` is not being used anymore, it might be necessary to look for other usages if it in the rest of the code.

Reply:
I don't think the variable was used before as well. I've removed this line.

Reply:
The only reference I can find about app.env is in a sentry configuration instruction. I'm not sure how to amend that.

Reply:
A quick check (`git blame` I guess, although I'm using a PyCharm IDE option) tells me that that line had been added quite recently, in https://github.com/FlexMeasures/flexmeasures/pull/716. @Ahmad-Wahid, can you pease weigh in on whether we can safely remove this line?

Reply:
I believe the line can be removed, because #920 stopped using `flask_env` - we introduced a dedicated config setting to decide whether to use this feature there.

Reply:
It's okay to drop it, we have replaced it with `FLEXMEASURES_ENFORCE_SECURE_CONTENT_POLICY`.